modules/administration/pages/infra-maintenance.adoc

```diff
@@ -117,9 +117,9 @@ To avoid this always update the [package]``salt-minion`` package when available.
 
 == Inter-Server Synchronization Slave Server
 
-If you are using an inter-server synchronization slave server, update it after the {productname} Server update is complete.
+If you are using an Inter-Server Synchronization slave server, update it after the {productname} Server update is complete.
 
-For more in inter-server synchronization, see xref:administration:iss.adoc[].
+For more in Inter-Server Synchronization, see xref:administration:iss.adoc[].
```

Comment:
I'm not sure.  maybe, we should check other places where this appears.  In the not too far past we tried to avoid capitalization as much as possible.